packages/components/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 -   Make the `Popover.Slot` optional and render popovers at the bottom of the document's body by default. ([#53889](https://github.com/WordPress/gutenberg/pull/53889)).
 -   `ProgressBar`: Add transition to determinate indicator ([#53877](https://github.com/WordPress/gutenberg/pull/53877)).
 -   Prevent nested `SlotFillProvider` from rendering ([#53940](https://github.com/WordPress/gutenberg/pull/53940)).
+-   Making Circular Option Picker a `listbox`. Note that while this changes some public API, new props are optional, and currently have default values; this will change in another patch. ([#52255](https://github.com/WordPress/gutenberg/pull/52255))
```

Comment:
What are we planning on changing exactly in a future patch?

Reply:
Sort of [touched on this above](#discussion_r1312998526), but ultimately all consumers should be providing their own label for the picker. `CircularOptionPicker` now requires an `aria-label` or `aria-labelledby`, and while `ColorPalette`, `DuotonePicker` and `GradientPicker` can all pass them through, they also provide a default value to obfuscate the need. Eventually they too will require one or the other, and not have a default; this will be addressed in #54055.